ig-data/input/pagecontent/mapping_guidance.md

```diff
@@ -1,48 +1,63 @@
+<style type="text/css">
+    .tg  {font-size:90%;border-collapse:collapse;border-spacing:0;border-color:#ccc;margin-left:auto; margin-right:auto; width:80%}
+    .tg td{padding:10px 5px;border-style:solid;border-width:0px;overflow:hidden;word-break:bold;border-top-width:1px;border-bottom-width:1px;border-color:#ccc;color:#333;background-color:#fff;text-align:center;vertical-align:middle}
+    .tg th{font-weight:bold;padding:10px 5px;border-style:solid;border-width:0px;overflow:hidden;word-break:normal;border-top-width:1px;border-bottom-width:1px;border-color:#ccc;color:#333;background-color:#f0f0f0;text-align:center;vertical-align:middle}
+    .tg caption{padding:10px 5px; font-weight:bold}
+    .tg .tg-bold{font-weight:bold;border-color:inherit;text-align:center;vertical-align:middle}
+    .tg .tg-altbck{background-color:#f9f9f9;border-color:inherit;text-align:center;vertical-align:middle}
+    .tg .tg-altbck-bold{background-color:#f9f9f9;font-weight:bold;border-color:inherit;text-align:center;vertical-align:middle}
+    .center {
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+}
+    </style>
+
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir ../../input-cache/schemas-r5/fhir-single.xsd">
 
-<h1>ICAREdata&reg; Structured Phrases to mCODE&trade; DSTU2 FHIR Mapping Guidance</h1>
+<h1>ICAREdata&reg; Structured Phrases to mCODE&trade; Mapping Guidance</h1>
 
 <p>There are currently two structured phrases, including:</p>
-    <ul>
-      <li><strong>Cancer Disease Status</strong>, which maps to the mCODE CancerDisease Status profile; and</li>
-      <li><strong>Cancer Treatment Plan Change</strong>, which maps to the CarePlanWithReview profile defined in this implementation guide.</li>
-    </ul>
-<p>The mapping details for each strcutured phrase are provided below, and are aligned with mCODE v0.9.1 (FHIR DSTU2).</p>
+  <ul>
+    <li><strong>Cancer Disease Status</strong>, which maps to the mCODE CancerDiseaseStatus profile; and</li>
+    <li><strong>Cancer Treatment Plan Change</strong>, which maps to the CarePlanWithReview profile defined in this implementation guide.</li>
+  </ul>
+<p>The mapping details for each structured phrase are provided below, and are aligned with mCODE v1.0 FHIR R4.</p>
 
 <h2>Cancer Disease Status</h2>
 
   <p>The Cancer Disease Status structured phrase maps to the mCODE CancerDiseaseStatus profile. Mapping details are provided in Table 1 below. </p>
-  <p>For more information on the CancerDiseaseStatus profile, please consult the <a href="http://standardhealthrecord.org/guides/mcode/" target="_blank">mCODE 0.9.1 DSTU2 specification</a>.</p>
+  <p>For more information on the CancerDiseaseStatus profile, please consult the <a href="http://standardhealthrecord.org/guides/mcode/" target="_blank">mCODE specification</a>.</p>
```

Comment:
Change link to published mCODE specification. The SHR link will not be supported. The link (available soon) will be: https://hl7.org/fhir/us/mcode/STU1